### ISSUE TITLE
Wildcard commands aren't auto-completing

### ISSUE THREAD

**Issue by pcgeek86** [PowerShell/vscode-powershell#153]
In the PowerShell ISE, I frequently type partial command names, containing wildcards. For example, to find the command `New-AzureRmVirtualNetworkSubnetConfig`, I might type `*subnet*` and hit CTRL + SPACE to invoke Intellisense.

Unfortunately, this isn't currently working in the Visual Studio Code PowerShell Extension.

![image](https://cloud.githubusercontent.com/assets/466713/14697226/1876de00-073f-11e6-9ba6-715fd55d5cca.png)

![image](https://cloud.githubusercontent.com/assets/466713/14697235/275dd63a-073f-11e6-8410-da7799a1ad43.png)

![image](https://cloud.githubusercontent.com/assets/466713/14697241/321c8be8-073f-11e6-818b-0cd2fc336762.png)

![image](https://cloud.githubusercontent.com/assets/466713/14697249/4000565e-073f-11e6-9a72-2757f7305d86.png)

Cheers,
**Trevor Sullivan**
https://trevorsullivan.net
https://twitter.com/pcgeek86

**Comment by daviwil** [comment]
I'll be that this is an issue on VS Code's side.  They add some extra "intelligence" when it comes to gathering completions at the cursor location so they might be throwing out the results that come back from the language server in this case.  I'll look into it!

**Comment by daviwil** [comment]
Moving to 0.7.0, definitely looks like an issue on VS Code's side.  I see that IntelliSense results are coming back for wildcard completions but they aren't being displayed.  Will file a bug at their repo to track this.

**Comment by pcgeek86** [comment]
@daviwil Thanks for discovering the root cause! Hoping they can fix this soon.

**Comment by pcgeek86** [comment]
@daviwil Any update on this? I'm still encountering this bug. Thanks!

**Comment by daviwil** [comment]
Nah, too many other high priority issues recently.  I'll assign it to the next feature update in case we get a chance to look into it.

**Comment by pcgeek86** [comment]
@tylerl0706 If we could figure out what's going on here, this would help bring the VSCode development experience in line with the console experience. Let me know if I can help clarify the issue at all.

**Comment by tylerl0706** [comment]
@daviwil did you ever get the chance to open that issue on VSCode?

**Comment by TylerLeonhardt** [comment]
I did https://github.com/microsoft/vscode/issues/92433

**Comment by JustinGrote** [comment]
@TylerLeonhardt your dependent issue was closed, asking you to ask over at the extensions slack for help on it (strangely)

**Comment by SydneyhSmith** [comment]
Yep, we need to implement this but unfortunately the platform doesnt give us a good mechanism yet.

**Comment by JustinGrote** [comment]
I delved some more and found the filter code in vscode itself, if I get some time I may attempt a fix PR, but yeah, I wouldn't expect this anytime soon. I don't think it happens with other editors (vim, etc.) though.
